docs/EnterpriseScale-Roadmap.md

```diff
@@ -11,6 +11,8 @@ We intend to update the content within this repo in alignment with Azure Semeste
 | Contoso Reference Implementation | End to end reference implementation to supplement Architecture and Design recommendations. | Complete (April, 2020) |
 | Publish Enterprise-Scale GitHub Action in Actions marketplace | This enables Resource discovery, deployments and operationalize IaC. | Complete (June, 2020) |
 | Azure DevOps Support | Provide ability to operate AzOps within Azure DevOps with Azure Pipelines. | Complete (August, 2020) |
-| Additional reference implementations | Prescriptive first-party reference implementation for the Enterprise-Scale architecture for different enterprise scenarios and size | In progress (November, 2020) |
+| Additional reference implementations | Prescriptive first-party reference implementation for the Enterprise-Scale architecture for different enterprise scenarios and size | Complete (October, 2020) |
 | Workload Specific landing zones in Enterprise-Scale | AKS, WVD, SAP, HPC, Analytics <br/> (Seeking community Contribution) |  Planned |
 | Hybrid Management in Enterprise-Scale landing zones | Azure Arc | Planned |
+| Support for N regions | ES Reference Implementations (Contoso, Adventure Works) | January, 2021 |
+| Support for connecting N landing zones | ES Reference Implementations (Contoso, Adventure Works) | January, 2021 |
```

Comment:
@krnese @juazasan is there anything here that would prevent adding Trey Research to the list?

Reply:
Haven't seen that Adventure works already added support from connecting the Corp Landing Zones and which is also I also had in mind for Trey Research. I am not able to see any potential blockers to add similar capabilities in Trey Research. That said we agreed on closing this PR as it is now.